types/dom-speech-recognition/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+	    "dom"
```

Comment:
Indentation looks off here.

Reply:
Right, somehow I put in there a tab instead of spaces. Fixed.